Marlin/Configuration.h

```diff
@@ -118,7 +118,7 @@
 // The following define selects which electronics board you have.
 // Please choose the name from boards.h that matches your setup
 #ifndef MOTHERBOARD
-  #define MOTHERBOARD BOARD_RAMPS_14_EFB
+  #define MOTHERBOARD BOARD_RAMPS4DUE_EFB
```

Comment:
OK for the 32 bit branch, but shouldn't be merged to the main branch...

@thinkyhead What do you think?